src/data/teams.json

```diff
@@ -124,8 +124,8 @@
     "platforms": [
       {
         "name": "github",
-        "label": "thenewboston-python",
-        "link": "https://github.com/thenewboston-developers/thenewboston-python"
+        "label": "Community",
```

Comment:
> with "thenewboston-Management/Community Tasks" as the reference name.

I think the label here needs to be written like repo name as mentioned.

@ma2b0043, correct?

Reply:
Nope, ignore this. My bad.